src/main/java/io/dropwizard/flyway/cli/DbInfoCommand.java

```diff
@@ -1,23 +1,25 @@
 package io.dropwizard.flyway.cli;
 
-import io.dropwizard.flyway.FlywayConfiguration;
 import io.dropwizard.Configuration;
 import io.dropwizard.db.DatabaseConfiguration;
+import io.dropwizard.flyway.FlywayCommand;
+import io.dropwizard.flyway.FlywayCommands;
+import io.dropwizard.flyway.FlywayConfiguration;
 import net.sourceforge.argparse4j.inf.Namespace;
 import org.flywaydb.core.Flyway;
 
-import static org.flywaydb.core.internal.info.MigrationInfoDumper.dumpToAsciiTable;
-
 public class DbInfoCommand<T extends Configuration> extends AbstractFlywayCommand<T> {
+
+    private static final FlywayCommand COMMAND = FlywayCommand.INFO;
+
     public DbInfoCommand(final DatabaseConfiguration<T> databaseConfiguration,
                          final FlywayConfiguration<T> flywayConfiguration,
                          final Class<T> configurationClass) {
-        super("info", "Prints the details and status information about all the migrations.",
-                databaseConfiguration, flywayConfiguration, configurationClass);
+        super(COMMAND, databaseConfiguration, flywayConfiguration, configurationClass);
     }
 
     @Override
     public void run(final Namespace namespace, final Flyway flyway) throws Exception {
-        System.out.println(dumpToAsciiTable(flyway.info().all()));
```

Comment:
Since this is more or less considered a library to dropwizard, we should not be printing via `System.out.println()`.

References:
[Why using System.out.println() so bad?](https://programmers.stackexchange.com/questions/161194/why-is-using-system-out-println-so-bad)
[Do not use System.out.println in server side code](https://stackoverflow.com/questions/8601831/do-not-use-system-out-println-in-server-side-code)